frontend/src/reducers/varianceReducer.js

```diff
@@ -12,11 +13,17 @@ const varianceReducer = (state = initialState, action) => {
         ...state,
         mineVariances: action.payload.records,
       };
+    case actionTypes.STORE_VARIANCE:
+      return {
+        ...state,
+        variance: action.payload,
+      };
     default:
       return state;
   }
 };
 
 export const getMineVariances = (state) => state[VARIANCES].mineVariances;
+export const getVariance = (state) => state[VARIANCES].variance;
```

Comment:
Purely for my own education:  Why is VARIANCES in all caps?   I s the re a reason for that convention?

Reply:
Redux naming conventions, reducerTypes, and actionTypes are `SCREAMING_SNAKE_CASE`